<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoRpm</class>
 <widget class="QMainWindow" name="ArduinoRpm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>605</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>RPM Counter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Speed-48.png</normaloff>Speed-48.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#ArduinoRpm {
	background: #f5f4f4;
}
#pushButton {
 background: #55ff55;
  background-image: -webkit-linear-gradient(top, #55ff55, #84f084);
  background-image: -moz-linear-gradient(top, #55ff55, #84f084);
  background-image: -ms-linear-gradient(top, #55ff55, #84f084);
  background-image: -o-linear-gradient(top, #55ff55, #84f084);
  background-image: linear-gradient(to bottom, #55ff55, #84f084);
  border-radius: 6px;
  font-family: Georgia;
  color: #4d4b4d;
  font-size: 16px;
  padding: 10px 20px 10px 20px;
  text-decoration: none;
}

#pushButton:hover:!pressed {
  background: #03e803;
  background-image: -webkit-linear-gradient(top, #03e803, #0fdb0f);
  background-image: -moz-linear-gradient(top, #03e803, #0fdb0f);
  background-image: -ms-linear-gradient(top, #03e803, #0fdb0f);
  background-image: -o-linear-gradient(top, #03e803, #0fdb0f);
  background-image: linear-gradient(to bottom, #03e803, #0fdb0f);
  text-decoration: none;
}

#statusBar {
background: #373b41;
color: #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Dec</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>RPM :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionLCD_Output"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuShow"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLCD_Output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LCD Output</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
